boot/zephyr/CMakeLists.txt

```diff
@@ -63,7 +63,15 @@ set(NRF_DIR "${MCUBOOT_DIR}/ext/nrf")
 
 if(CONFIG_BOOT_USE_NRF_CC310_BL)
 set(NRFXLIB_DIR ${ZEPHYR_BASE}/../nrfxlib)
-assert_exists(NRFXLIB_DIR)
+if(NOT EXISTS ${NRFXLIB_DIR})
+  message(FATAL_ERROR "
+    ------------------------------------------------------------------------
+    No such file or directory: ${NRFXLIB_DIR}
+    Current configuration enables nRF CC310 crypto accelerator hardware
+    by `BOOT_USE_NRF_CC310_BL` option. Please follow `ext/nrf/README.md` guide
+    to fix your setup.
```

Comment:
"... fix your setup, or disable the HW accelerator with `-DCONFIG_BOOT_ECDSA_TINYCRYPT=y`."

Reply:
Maybe we could even write:

fix your setup or use tinycrypt instead of the HW accelerator. To use the tinycrypt set `CONFIG_BOOT_ECDSA_TINYCRYPT` to `y`.